seed/models/analyses.py

```diff
@@ -43,7 +45,7 @@ class Analysis(models.Model):
     service = models.IntegerField(choices=SERVICE_TYPES)
     start_time = models.DateTimeField(null=True, blank=True)
     end_time = models.DateTimeField(null=True, blank=True)
-    status = models.IntegerField(choices=STATUS_TYPES)
+    status = models.IntegerField(default=PENDING_CREATION, choices=STATUS_TYPES)
```

Comment:
Setting a default for creating an analysis seemed like something we should do b/c 99% of the time it should be in its initial "state"